tidb-cloud/branch-overview.md

```diff
@@ -35,12 +35,12 @@ You can create branches easily and quickly to get isolated data environments. Br
 
 Currently, TiDB Cloud Serverless branches are in beta and free of charge.
 
-- You can only create branches for TiDB Cloud Serverless clusters that are created after July 5, 2023.
-
 - For each organization in TiDB Cloud, you can create a maximum of five TiDB Cloud Serverless branches by default across all the clusters. The branches of a cluster will be created in the same region as the cluster, and you cannot create branches for a throttled cluster or a cluster larger than 100 GiB.
 
 - For each branch of a free cluster, 10 GiB storage is allowed. For each branch of a scalable cluster, 100 GiB storage is allowed. Once the storage is reached, the read and write operations on this branch will be throttled until you reduce the storage.
 
+If tables in your cluster have TiFlash replicas, these replicas will be temporarily unavailable after you create a new branch, because the replica data needs to be rebuilt in TiFlash.
```

Comment:
```suggestion
- If tables in your cluster have TiFlash replicas, these replicas will be temporarily unavailable after you create a new branch, because the replica data needs to be rebuilt in TiFlash.
```